tests/emukit/core/test_optimization.py

```diff
@@ -40,3 +47,49 @@ def test_multi_source_acquisition_optimizer():
     opt_x, opt_val = optimizer.optimize(acquisition)
     np.testing.assert_array_equal(opt_x, np.array([[0., 1.]]))
     np.testing.assert_array_equal(opt_val, np.array([[2.]]))
+
+
+def test_random_search_acquisition_optimizer():
```

Comment:
I think this file is poorly named. Feel free to rename and create separate ones for different optimizers

Reply:
They are now all separated.